### DISCUSSION TITLE
Looking for new writer for Kiali feature updates (Medium Blog)

### DISCUSSION THREAD

**Discussion by israel-hdez** [kiali/kiali#4281] [General]
Hi, all.

I just created the following ticket to track progress about writing the next Kiali feature update in Medium: https://github.com/kiali/kiali/issues/4280. As you see, this will cover 6 versions.

I've been writing "feature update" posts for slighly more than 2 years. The first one I wrote was for Sprint 18, when Kiali 0.15.0 was released. In case you want to read my very first post and see a few images of the old Kiali UI, here is the link: https://medium.com/kialiproject/kiali-sprint-18-wrap-up-52d1502adf61.

In the beggining, I tried to write a blog post each time a release was published. But it's hard to keep that pace and several  times I had to write a post covering more than one version. As a result, I only have wrote 20 posts, plus the upcoming one :-)

Now, after 2 years of writing, I'll will be doing one last post and stopping. The main reason is that writing these posts requires a lot of focus, which lately has been hard to keep during a day, so the time spent on writing increases. Since writing is not my main priority, I think this time can be best used on other more important tasks. Secondarily, my interest on writing these feature updates have been decreasing and *I think* (I'm not sure) that the reason behind this is that I have learned whatever I needed to learn from writing these posts, and this kind of writing is probably no longer bringing me more personal growth.

So, I'm looking for somebody else who would like to give continuity to these Medium posts. If somebody is interested, please drop a message. If nobody volunteers, I'll simply use my last post to announce that there will be no other feature updates and will defer community to reading [release notes](https://kiali.io/news/release-notes/), and watching [recordings of sprint demos on YouTube](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) to gain insight about changes on each release.

BTW, this doesn't mean that I will completly stop writing in Medium. I'll probably write other kind of posts, like explaining specific features as others have done, but it is going to be more sporadic.

**Comment by abonas** [comment]
@israel-hdez thank you for all the contributions so far and also for articulating so greatly about how you feel now and the next steps. 
I suggest to loop in @xeviknal to tweet about help is needed in this area, as well as mentioning in next demo, and perhaps label this as docs and as good first issue? Because writing about new features can go hand in hand with learning how they work in Kiali to be able to describe them well in writing :)

**Reply by xeviknal** [reply]
Agree! Let's point to the community to this discussion and find a successor for this.

**Reply by israel-hdez** [reply]
I already saw your tweets, @xeviknal! Thanks for covering on that social network :-)

And I'll make sure to add a slide in the next demo. Nice suggestion, @abonas!
I'll wait to see if there is a volunteer. If nobody volunteers at the time I finish my last post, then I'll create an issue so that we can archive this GitHub-discussion.

**Comment by xeviknal** [comment]
I completely understand your position. And I think that the only thing I can say is thank you for the good posts giving visibility of team's work!

**Reply by israel-hdez** [reply]
Thanks! I tried to do my best at giving awareness of our beloved Kiali project ;-)

**Comment by hhovsepy** [comment]
@israel-hdez I can take this responsibility of writing Medium blog posts about Kiali releases. Even though I do not have any blogging experience, but will give a try.

**Reply by israel-hdez** [reply]
Thanks @hhovsepy !
I suggest to check the blog: https://medium.com/kialiproject and find my posts. Give them a quick "scan" to have an idea of the structure. Clearly, you don't need to follow that same structure or style. Use your own style and format :-)

If you are confident that you will be giving continuity, in my last post I can mention you to give clarity that you will be the author of new written feature updates. If not, I can simply say that I will no longer write these posts, but that probably somebody else will do. What would be your preference?

**Reply by hhovsepy** [reply]
Hi @israel-hdez , yes please mention me as a new author, @haik.hovsepian is my personal account there.
